src/components/SpinnerLoading/index.jsx

```diff
@@ -3,9 +3,9 @@ import ReactLoading from 'react-loading';
 import theme from '../../styles/theme';
 import { Container } from './styles';
 
-export default function SpinnerLoading() {
+export default function SpinnerLoading({ top = '45%', right = '30%' }) {
```

Comment:
Its not good practice to pass styling properties to component instead we have to do it in component itself and if we need to pas somthing we have to make it informall for example: size : string = huge ,smale . . .